packages/@ourworldindata/types/src/gdocTypes/Gdoc.ts

```diff
@@ -54,7 +54,21 @@ export interface OwidGdocBaseInterface {
 
 export interface OwidGdocPostInterface extends OwidGdocBaseInterface {
     content: OwidGdocPostContent
-    linkedDocuments?: Record<string, OwidGdocPostInterface>
+    linkedDocuments?: Record<string, OwidGdocMinimalPostInterface>
+}
+
+// Used for linkedDocuments attachments, instead of attaching the entire gdoc model
+export interface OwidGdocMinimalPostInterface {
+    id: string
+    title: string
+    slug: string
+    authors: string[]
+    publishedAt: string
+    published: boolean
+    subtitle: string
+    excerpt: string
+    type: OwidGdocType
+    "featured-image"?: string
 }
```

Comment:
I find [the justifications you wrote for lightning update props](https://github.com/owid/owid-grapher/blob/a4c1c87335821bcdbb69251111cb42440d7608f1/adminSiteClient/gdocsDeploy.ts#L59-L66) useful. Maybe we could do something similar here?